examples/gallery/embellishments/gmt_logo.py

```diff
@@ -1,6 +1,6 @@
 """
-Logo
-====
+GMT logo
+========
 
 The :meth:`pygmt.Figure.logo` method allows to place the GMT logo on a map.
```

Comment:
xref: https://github.com/GenericMappingTools/pygmt/issues/2437. We can use either `figure` or `plot` here, but not `map`.
```suggestion
The :meth:`pygmt.Figure.logo` method allows to place the GMT logo on a figure.
```